### PR TITLE
Style homepage

### PR DESCRIPTION
**Pull request by Maenhout**
- Changing the title style h1
- Changing the style of the navbar
- Adding the background

### CODE REVIEW

**Review by JulienDelemarle — DISMISSED** [review]
CSS , background add and App.jsx changes: ok

**Review by JulienDelemarle — DISMISSED** [review]
add typo link ok
css modification ok

**Review by JulienDelemarle — DISMISSED** [review]
typo modification ok

File: src/App.jsx [hunk 1]
```diff
@@ -1,22 +1,30 @@
...
-            <Home />
-          </Route>
-          <Route path="/quiz">
-            <QuizPage />
-          </Route>
-        </Switch>
-      </Router>
-    </div>
+    <SApp>
+      <div className="App">
```
Comment (LoicBrassart) [inline src/App.jsx]: Does this `<div>` serve any purpose ?

**Review by LoicBrassart — COMMENTED** [review]
Saw no Evil !

**Review by JulienDelemarle — DISMISSED** [review]
delete unused div

**Review by JulienDelemarle — APPROVED** [review]
updates ok

**Review by LoicBrassart — APPROVED** [review]
Saw no Evil

Note: 
With our current workflow, the feature should be *finished* by the time the first review comes into place. As a consequence, when a review asks for a fix, there should be (0..n) commits after it to fix what was asked and **only that** : do not keep adding unrelated code, pretty please, or it messes up code reviewers ;-)

**Pull request merged**